Source/Flow/Public/Nodes/FlowNode.h

```diff
@@ -82,52 +59,18 @@ class FLOW_API UFlowNode : public UObject, public IVisualLoggerDebugSnapshotInte
 	virtual void PostLoad() override;
 	// --
 
-	// Opportunity to update node's data before UFlowGraphNode would call ReconstructNode()
-	virtual void FixNode(UEdGraphNode* NewGraphNode);
-
 	virtual EDataValidationResult ValidateNode() { return EDataValidationResult::NotValidated; }
 
-	// used when import graph from another asset
-	virtual void PostImport() {}
-#endif
-
-	UEdGraphNode* GetGraphNode() const { return GraphNode; }
-
-#if WITH_EDITOR
-	void SetGraphNode(UEdGraphNode* NewGraph);
-
-	virtual FString GetNodeCategory() const;
-	virtual FText GetNodeTitle() const;
-	virtual FText GetNodeToolTip() const;
-
-	// This method allows to have different for every node instance, i.e. Red if node represents enemy, Green if node represents a friend
-	virtual bool GetDynamicTitleColor(FLinearColor& OutColor) const;
-
-	EFlowNodeStyle GetNodeStyle() const { return NodeStyle; }
-
-	// Short summary of node's content - displayed over node as NodeInfoPopup
-	virtual FString GetNodeDescription() const;
 #endif
 
-protected:
-	// Short summary of node's content - displayed over node as NodeInfoPopup
-	UFUNCTION(BlueprintImplementableEvent, Category = "FlowNode", meta = (DisplayName = "Get Node Description"))
-	FString K2_GetNodeDescription() const;
-
 	// Inherits Guid after graph node
 	UPROPERTY()
```

Comment:
Only full UFlowNodes (not AddOns) are assigned GUIDs and are stored in the FlowAsset's node array.

This means they are also the only things saved in the save state code as well.

This may be something that changes in the future.